Sources/TokamakDemo/main.swift

```diff
@@ -37,6 +37,9 @@ let renderer = DOMRenderer(
     }
     VStack {
       Counter(count: 5, limit: 15)
+        .padding()
+        .background(Color(red: 0.9, green: 0.9, blue: 0.9, alpha: 1.0))
+        .border(Color.red, width: 3)
```

Comment:
Is there anything else in this PR that could be added to the demo to verify that things are working? (I know that absence of unit tests is not great, but I'm a bit blocked on resolving XCTest support on the toolchain side by https://github.com/swiftwasm/swift/issues/1330)

Reply:
cornerRadius is implemented but only partially working...